test/contract/bookit-api.ts

```diff
@@ -0,0 +1,9 @@
+import * as request from 'superagent'
+
+// const bookRoom = (name) => request.get(`https://api.github.com/users/${name}`);
+export const bookRoom = () => request
+  .post('https://integration-bookit-api.buildit.tools/v1/booking')
+  .send({ bookableId: '-1', bookingId: '-2', startDateTime: '2017-10-02T22:03:27.409Z', endDateTime: '2017-10-02T22:04:27.409Z', subject: 'Getting to know you' })
+  .set('Content-Type', 'application/json');
```

Comment:
Ohkay so...
1) probably want to parameterize the URL via environment variable?  See https://github.com/buildit/bookit-client-react/blob/master/test/end2end/features/step-definitions/booking-steps.js#L5

2) First, this is going to start failing pretty soon when '-1' and '-2' don't actually exist.  Second, this is going to create state/bookings on the server so the second time this executes, it's going to fail because it already exists.  I'm not saying we have to solve it now, but we need to be thinking about how we're going to manage this..